### DISCUSSION TITLE
Azure sFTP + FTPS how to ?

### DISCUSSION THREAD

**Discussion by esener303** [AzureCommunityPL/group#459] [Pomoc] [answered]
Cześć, 

Czy miał ktoś okazje tworzyć takie rozwiązanie w Azure ? korzystałem już z sFTP jako Azure Storage Account ale tak się zastawiam nad FTPS bo potrzebuje obu protokołów nie muszą oba sięgać do tego samego folderu / zbioru plików. 

Jakieś pomysły lub sugestie ?

**Comment by mgrabarz** [comment, marked as answer]
FTPS nie jest zbyt mocno "lubiany" przez dostawców chmurowych. W odróznieniu od SFTP wymaga otwarcia/natowania kilku portów, a ten od wymiany danych może być dynamicznie negocjowany w pewnym zakresie. Efekt jest taki, że SFTP jest wsperany na Azure Storage, a FTPS nie. 

Jedyna opcja na FTPS to VM/kontener z zainstalowanym odpowiednim softem i certyfikatem TLS. Jest też AppService, który wystawia FTPSa, ale tej opcji bym nie rozważał.

**Reply by esener303** [reply]
dzięki za wyjaśnienie, cieszę się że myślałem podobnie i pomysł z App Service FTPs odrzuciłem na starcie.
